app/src/test/java/com/ichi2/apisample/MusIntervalTest.java

```diff
@@ -47,6 +43,7 @@ public void checkExistence_NoStartingNotes() throws AnkiDroidHelper.InvalidAnkiD
 
         AnkiDroidHelper helper = mock(AnkiDroidHelper.class, new ThrowsExceptionClass(IllegalArgumentException.class));
         doReturn(modelId).when(helper).findModelIdByName(defaultModelName);
+        doReturn(true).when(helper).isModelValid(any(Long.class), any(String[].class));
```

Comment:
It probably makes sense to mock exact 'modelId' instead of 'any':
doReturn(true).when(helper).isModelValid(modelId, any(String[].class));